sys/kern/vfs_aio.c

```diff
@@ -2715,6 +2790,13 @@ aiocb32_fetch_error(struct aiocb *ujob)
 	return (fuword32(&ujob32->_aiocb_private.error));
 }
 
+static void
+aiocb32_free_kaiocb(struct kaiocb *kjob)
```

Comment:
Perhaps just use aiocb_free_kaiocb() directly for the free op for compat32?

Reply:
I waffled on that.  I'll go ahead and do it.